shared/middleware/shared_resources.rb

```diff
@@ -17,9 +17,11 @@ def self.set_max_age(type, default)
   ONE_HOUR = 3600
 
   configure do
+    # Note 1: pegasus/router.rb has additional configuration for Sass::Plugin
+    # Note 2: the generated css files written to /pegasus/cache/css are served
+    #         from the url path /shared/css (see route below)
```

Comment:
when I add a .scss file `pegasus/sites.v3/code.org/public/css/dave.scss`, then restart the local server and load a pegasus page, I see a .css file show up in `pegasus/sites.v3/code.org/public/css/generated/dave.css`. so far so good. However now I am wondering, how does `pegasus/cache/css` come into play? also, what url path would I use to access my new css file?

Reply:
The cache stores some kind of internal intermediate parsed version of the .scss file (with an extension of .scssc).  The official documentation is no longer available, but I found a copy and there's a little bit more information about caching [here](https://paulirish.github.io/Sass-reference/#caching).

The new css file can be accessed at `code.org/css/generated/dave.css`.